pkg/collector/corechecks/system/cpu/cpu_windows_test.go

```diff
@@ -0,0 +1,83 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build windows
+
+package cpu
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/metrics"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator/mocksender"
+	"github.com/stretchr/testify/mock"
+)
+
+var (
+	firstSample = []TimesStat{
+		{
+			CPU:    "cpu-total",
+			User:   1229386,
+			System: 263584,
+			Idle:   25496761,
+		},
+	}
+	secondSample = []TimesStat{
+		{
+			CPU:    "cpu-total",
+			User:   1229586,
+			System: 268584,
+			Idle:   25596761,
+		},
```

Comment:
Isn't the `Kernel` field missing from these test samples (and is therefore defaulting to 0, which is false since `Kernel = System + Idle` which isn't 0)?

Reply:
Done, but I was expecting this to affect the test results and it didn't... maybe there's something else that's not right.